test/Walker_spec.ts

```diff
@@ -33,4 +33,11 @@ describe('Walker', () => {
   it('should locate a dep of a dev dep that is also a top level prod dep as a prod dep', () => {
     expect(dep('debug')).to.have.property('depType', DepType.PROD);
   });
+
+  it('should locate a dep of a dev dep that is optional as a dev_optional dep', function () {
```

Comment:
arrow function like the rest?

Reply:
`this.skip` only works on non-arrow functions (needs to be bound correctly)